pages/Documents/RichMessaging/AppleMessagesForBusinessTemplates/form-message-template.md

```diff
@@ -0,0 +1,349 @@
+
+  For all possible options for page configuration check [Apple Documentation](https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message).
+
+#### Example Page Template
+
+```json
+{
+  "pageIdentifier": "1",
+  "type": "input",
+  "nextPageIdentifier": "2",
+  "title": "Your email address",
+  "subtitle": "Enter your email address",
+  "options": {
+      "required": true,
+      "inputType": "singleline",
+      "maximumCharacterCount": 250,
+      "keyboardType": "emailAddress",
+      "textContentType": "emailAddress"
+  }
+}
+```
+
+### Example Body Template
+The structured content body will allow you to define spalsh screen of the Business Form. 
+
+Following Properties must be included in Body template
+* type
+* tag
+* elements
+
+Every splash screen contains title, subtitle and button, these must be included in elements section.
```

Comment:
**Each** splash screen contains **a** title, subtitle and button. These must be included in **the** elements section.

Reply:
since there can only be one splash screen, let's rephrase to this:
A splash screen contains a title, subtitle and button. These must be included in the elements section.

Reply:
Implemented requested change